client/apps/game/src/services/api.ts

```diff
@@ -2,10 +2,10 @@ import { ContractAddress, HexPosition, ID, StructureType } from "@bibliothecadao
 import { getChecksumAddress } from "starknet";
 import { env } from "../../env";
 
-const API_BASE_URL = env.VITE_PUBLIC_TORII + "/sql";
+export const API_BASE_URL = env.VITE_PUBLIC_TORII + "/sql";
 
 // Define SQL queries separately for better maintainability
-const QUERIES = {
+export const QUERIES = {
```

Comment:
don't export and create a fetch function directly in this file, same as all other fetches